LondonBritishLibrary/orient/BLorient4611.xml

```diff
@@ -35,9 +35,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <title ref="LIT2701Dawit"/>
                             <msItem xml:id="ms_i1.1">
                                 <locus from="3r" to="86v"/>
-                                <title ref="LIT2000Mazmur"/>
```

Comment:
this should stay though

Reply:
markup of the `incipit `is not the same as `msItem`, these are two different things, `msItem `needs a `title`, `incipit `may have a `type `or an inner markup